app/globals.css

```diff
@@ -2,26 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+.hide-scrollbar::-webkit-scrollbar {
+  width: 0.25rem; /* Set the width to 0 */
 }
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    --foreground-rgb: 255, 255, 255;
-    --background-start-rgb: 0, 0, 0;
-    --background-end-rgb: 0, 0, 0;
-  }
-}
-
-body {
-  color: rgb(var(--foreground-rgb));
-  background: linear-gradient(
-      to bottom,
-      transparent,
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
+.hide-scrollbar::-webkit-scrollbar-thumb {
+  background-color: transparent; /* Hide the thumb */
```

Comment:
We shouldn't need these extra css styles, I wonder why they are required!

Reply:
Inorder to hide the scrollbar the predefined one wasn't able to solve